mac

```diff
@@ -137,36 +146,19 @@ brew "pipenv"
 brew "tfenv"
 
 # Databases
-brew "postgres", restart_service: :changed
+brew "postgresql@14", restart_service: :changed
 brew "redis", restart_service: :changed
 
 # Development tools
 brew "awscli"
-brew "heroku/brew/heroku"
 brew "getsentry/tools/sentry-cli"
-
-# Applications
-cask "google-chrome"
-cask "firefox"
-cask "slack"
-cask "iterm2"
-cask "sublime-text"
-cask "sublime-merge"
-cask "visual-studio-code"
-cask "insomnia"
-cask "flycut"
-cask "bitwarden"
-cask "caffeine"
-cask "docker"
-cask "zoom"
-cask "tableplus"
```

Comment:
This is a pretty well-supported pattern (https://github.com/Homebrew/homebrew-cask; 20k stars on GitHub), but you're right that it's not the installation method that the application developers themselves intended.

I like the convenience of these lines for setting up a new laptop, but that's probably also because I often manage applications with `cask` locally anyway.

Totally fine if we want to pull this out of the file and have folks manage applications independently!

Reply:
I'm ok removing them to simplify and narrow the scope of this script, but it could be nice to just run the script and have a decent number of the apps you need, w/o having to add "install slack... etc." to our other setup docs.

Reply:
I think I'll leave it in the script but commented out.